setup.py

```diff
@@ -351,7 +351,12 @@ def get_extensions():
 
     ffmpeg_exe = distutils.spawn.find_executable('ffmpeg')
     has_ffmpeg = ffmpeg_exe is not None
-    if sys.platform != 'linux':
+    # Building torchvision with ffmpeg on MacOS or with Python 3.9 causes
+    # crash. See the following GitHub issues for more details.
+    # https://github.com/pytorch/pytorch/issues/65000
+    # https://github.com/pytorch/vision/issues/3367
+    if sys.platform != 'linux' or (
+            sys.version_info.major == 3 and sys.version_info.minor == 9):
```

Comment:
BTW I think we can remove the ffmpeg version checks below as well, right?

Reply:
https://github.com/pytorch/vision/pull/4413 does that. After merging this PR, #4413 should be good to merge.